source/hardware/controllers/hercules_djcontrol_jogvision.rst

```diff
@@ -0,0 +1,138 @@
+Hercules DJControl Jogvision
+============================
+
+-  `Manufacturer's product page <https://www.hercules.com/en-us/product/djcontroljogvision-2old/>`__
+-  `Manufacturer's support and downloads page <https://support.hercules.com/en/product/djcontroljogvision-en/>`__
+-  `Forum thread <https://www.mixxx.org/forums/viewtopic.php?f=7&t=12580>`__
+  
+Whether you’re just getting started DJing or you’ve already refined your skills, you can let your creativity run wild.
+Enjoy excellent precision and comfort to create your mixes using the circular displays on the jog wheels to guide your movements and perfect your scratches,
+and have fun with the amazing contactless AIR Control sensor.
+Your adventure starts right here and now. It’s your turn\!
+
+Compatibility
+-------------
+
+This controller is a class compliant USB MIDI and audio device, so it
+can be used without any special drivers on GNU/Linux, Mac OS X, and
+Windows. However, if you wish to use the `ASIO sound
+API <https://mixxx.org/manual/latest/en/chapters/preferences.html?highlight=asio#windows>`__
```

Comment:
Please replace all links to the manual with internal cross references:

    :ref:`Some text <some-cross-reference>`.

The cross-references are defined in the respective file *above* the headline you want to link to like this: `.. _some-coss-reference:`. See this for details:  https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html

Otherwise these links won't work as intended when generating a non-web manual (PDF, Qt Help, etc.).